src/structures/ApplicationCommand.js

```diff
@@ -156,6 +156,76 @@ class ApplicationCommand extends Base {
     return this.manager.delete(this, this.guildId);
   }
 
+  /**
+   * Whether this command equals another command. It compares all properties, so for most operations
+   * it is advisable to just compare `command.id === command2.id` as it is much faster and is often
+   * what most users need.
+   * @param {ApplicationCommand|ApplicationCommandData|APIApplicationCommand} command The command to compare with
+   * @returns {boolean}
+   */
+  equals(command) {
+    // If given an id, early return
```

Comment:
Need to describe why it could early return

```suggestion
    // If given an id, check if the id matches
```